tests/cli/commands/test_connection_command.py

```diff
@@ -111,6 +112,112 @@ def test_cli_connections_include_secrets(self):
             connection_command.connections_list(args)
 
 
+class TestCliExportConnections(unittest.TestCase):
```

Comment:
It would be valuable to test also for emergency situations when an operation fails, or the user gives an incorrect value as a parameter.

Reply:
Yes, this was just a draft to confirm the functionality. Will be adding the tests for all scenarios.